Test.ps1

```diff
@@ -9,6 +9,18 @@ if($LASTEXITCODE -ne 0) {
     throw "*ERROR**: failed to initialize Python Virtual Environment"
 }
 
+# Rename all project manifests to correct format.
+# This is done to avoid security scanning them for deprecated/vulnerable library dependencies
+# These sample projects are not intended to be used in production, but are used for testing
+Rename-Item -Path "./sample-projects/dotnet-samples/sample_csproj_PLACEHOLDER" -NewName "sample.csproj"
+Rename-Item -Path "./sample-projects/go-samples/compatible/go_mod_PLACEHOLDER" -NewName "go.mod"
+Rename-Item -Path "./sample-projects/go-samples/incompatible/go_mod_PLACEHOLDER" -NewName "go.mod"
+Rename-Item -Path "./sample-projects/java-samples/pom_xml_PLACEHOLDER" -NewName "pom.xml"
+Rename-Item -Path "./sample-projects/node-samples/package_json_PLACEHOLDER" -NewName "package.json"
+Rename-Item -Path "./sample-projects/python-samples/compatible/requirements_txt_PLACEHOLDER" -NewName "requirements.txt"
+Rename-Item -Path "./sample-projects/python-samples/incompatible/requirements_txt_PLACEHOLDER" -NewName "requirements.txt"
+Rename-Item -Path "./sample-projects/ruby-samples/Gemfile_PLACEHOLDER" -NewName "Gemfile"
+
```

Comment:
Should we rename them back after the test completes both so that the test can run correctly multiple times in a row and also so old dependencies are not hanging around on the file system?

Reply:
Discussed offline, going with a unique named temporary directory approach to enable devs to test any number of times.